get_data_meteorological_multiworker.py

```diff
@@ -0,0 +1,249 @@
+import os
```

Comment:
nitpick: imports aren't ordered. It doesn't take away the content of the code. If you want a place to start, you can look at the format [here](https://pypi.org/project/isort/)